MovieApp/PresentationLayer/Views/HomeScreen/HomeScreenViewController+Design.swift

```diff
@@ -16,17 +16,33 @@ extension HomeScreenViewController: ConstructViewsProtocol {
     func createViews() {
         navigationView = NavBarView()
         view.addSubview(navigationView)
+
+        movieCollection = createCollectionView()
+        view.addSubview(movieCollection)
+    }
+
+    private func createCollectionView() -> UICollectionView {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .vertical
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        return collectionView
     }
     
     func styleViews() {
         view.backgroundColor = .white
+        movieCollection.backgroundColor = .none
     }
     
     func defineLayoutForViews() {
         navigationView.snp.makeConstraints {
             $0.top.centerX.width.equalToSuperview()
             $0.height.equalTo(NavBarView.defaultHeight)
         }
+        
+        movieCollection.snp.makeConstraints {
+            $0.top.equalTo(navigationView.snp.bottom)
+            $0.width.bottom.equalToSuperview()
```

Comment:
It is better to define this as `$0.leading.trailing.bottom.equalToSuperview()`. Use edge constraints before width/height constraints, where applicable.